src/components/RecipeList.tsx

```diff
@@ -1,13 +1,21 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import ListGroup from "react-bootstrap/ListGroup";
 import Spinner from "react-bootstrap/Spinner";
 import Collapse from "react-bootstrap/Collapse";
 import Alert from "react-bootstrap/Alert";
 import FullRecipe from "./FullRecipe/FullRecipe";
-import { useGetAllRecipesQuery } from "../services/RecipeService";
+import { useGetAllRecipesMutation } from "../services/RecipeService";
 
 const RecipeList: React.FC = () => {
-  const { data: recipes, error, isLoading } = useGetAllRecipesQuery();
+  const [getAllRecipes, { data: recipes, isLoading, isError }] =
+    useGetAllRecipesMutation();
+
+  useEffect(() => {
+    getAllRecipes({
+      action: "get_file",
+      fileName: "base.json",
```

Comment:
base.json is taken in a correct way
but recipes not eg when I want to expand one recipe, app queries:
https://ustka.travel/api/63a2a85d15ab31599e2192aa [see screenshot I provided via skype]